cliquet/tests/test_authentication.py

```diff
@@ -79,3 +80,28 @@ def test_set_expires_the_value(self):
         time.sleep(0.1)
         retrieved = self.cache.get('foobar')
         self.assertIsNone(retrieved)
+
+
+class BasicAuthenticationPolicyTest(unittest.TestCase):
+    def setUp(self):
+        self.policy = authentication.BasicAuthAuthenticationPolicy()
+        self.request = DummyRequest()
+        self.request.headers['Authorization'] = 'Basic bWF0Og=='
+        self.request.registry.settings['cliquet.basic_auth_enabled'] = True
+
+    def test_prefixes_users_with_basicauth(self):
+        user_id = self.policy.unauthenticated_userid(self.request)
```

Comment:
you mean the attribute on the request ?

It is set elsewhere in pyramid I guess....